src/status_im/contexts/profile/login/events.cljs

```diff
@@ -152,15 +152,12 @@
 
 (rf/reg-event-fx
  :profile.login/login-node-signal
- (fn [{{:onboarding/keys [recovered-account? new-account?] :as db} :db}
-      [{:keys [settings account ensUsernames error]}]]
+ (fn [{db :db} [{:keys [settings account ensUsernames error]}]]
    (log/debug "[signals] node.login" "error" error)
    (if error
      {:db (update db :profile/login #(-> % (dissoc :processing) (assoc :error error)))}
      {:db (dissoc db :profile/login)
-      :fx [(when (and new-account? (not recovered-account?))
-             [:dispatch-later [{:ms 1000 :dispatch [:wallet-legacy/set-initial-blocks-range]}]])
-           [:dispatch-later [{:ms 2000 :dispatch [:ens/update-usernames ensUsernames]}]]
+      :fx [[:dispatch-later [{:ms 2000 :dispatch [:ens/update-usernames ensUsernames]}]]
            [:dispatch [:profile.login/login-existing-profile settings account]]]})))
```

Comment:
@status-im/wallet-mobile-devs 


Hi guys, I've seen this event multiple times being reported in re-frame logs as non-existent. I've removed it now, but I wonder if we attempted to do something else. wdyt?

Reply:
Good catch, @ulisesmac. That event was meant to call setInitialBlocksRange endpoint from status-go on the legacy wallet: https://github.com/status-im/status-mobile/blob/4a8a23a4c43b6d54bec98e8cd42d7c068738c3f8/src/status_im/wallet/core.cljs#L923-L937

But that endpoint was removed more than one year ago: https://github.com/status-im/status-go/pull/3953

So I would say it should be safe to remove these lines.